go/processor/floria/processor.go

```diff
@@ -70,31 +73,74 @@ func (p *processor) Run(
 		// Create new contract
 	} else {
```

Comment:
no else, if error you returned already.

Reply:
This `else` is not from an error check but from an if checking whether it is a create (not yet implemented) or a normal call.